scripts/proto_splitter/proto2mesh.py

```diff
@@ -449,18 +551,80 @@ def write_obj(self, meshes):
         default=None,
         help='Specifies the proto file, or a directory. Converts all .proto files, if it is a directory.',
     )
+    optParser.add_option(
+        '--output',
+        dest='outPath',
+        default=None,
+        help='Specifies the output path. Only for directory conversions.',
+    )
+    optParser.add_option(
+        '--mt', '--multithreaded',
+        dest='mt',
+        default=False,
+        action='store_true',
+        help='If set, enables multicore processing for directories with several PROTO files. May not work on Windows.',
+    )
+    optParser.add_option(
+        '--verbose', '-v',
+        dest='verbose',
+        default=False,
+        action='store_true',
+        help='If set, detailed output of mesh conversion is shown in console',
```

Comment:
consistency with other help lines:
```suggestion
        help='If set, detailed output of mesh conversion is shown in console.',
```